### PR TITLE
Mile Stone 1: Review Microverse Trial Project

### PR DESCRIPTION
**Pull request by Felix45**
# Survey App: Homepage
In this milestone I
- Added a homepage for the Survey App
- Added a HTML form for collecting survey responses

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,5 @@
-# survey-form
\ No newline at end of file
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```
Comment (Felix45) [inline README.md]: The readme file in your project is not descriptive enough, please use this template to document your project professionally [readme file template](https://github.com/microverseinc/readme-template)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    
+    <h1 id="title">Survey Form</h1>
```
Comment (Felix45) [inline index.html:25]: Consider wrapping the `h1` tag in an HTML5 `header` tag, this semantic tag will give clear meaning to web browsers and search engines of the heading on this page.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    </div>
+
+    <div>
+      <textarea name="comment" id="comment" cols="30" rows="10"></textarea>
+    </div>
+    <button type="submit" id="submit">submit</button>
+  </form>
+
+
+    <p>Hello World, we are software developers!!</p>
```
Comment (Felix45) [inline index.html:64]: Consider wrapping this section of the web page in a ` main ` tag. The HTML5 semantic tag ` main ` indicates to a web browser and search engine the most important part of your web page

**Review by Felix45 — COMMENTED** [review]
Hi @Felix45 

# Changes Required :recycle: 
- Please set up CSS, HTML, and JavaScript Linters on your project.
- Please check my comments below and make all required changes.

Submit for a re-review when you are done

Thanks :grin:

**Review by ShadyShawkat — APPROVED** [review]
## Hi @Felix45, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**